<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>details_dialog</class>
 <widget class="QDialog" name="details_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lookup Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title_label">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Lookup Details</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="file_path_line_edit"/>
   </item>
   <item>
    <widget class="QTreeWidget" name="tree_widget">
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Matches</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Line No</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Line Text</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="open_file_button">
     <property name="text">
      <string>Open File</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
